app/controllers/CannotUseRevisedUnloadingProcedureController.scala

```diff
@@ -41,9 +48,16 @@ class CannotUseRevisedUnloadingProcedureController @Inject() (
       }
 
   def onSubmit(arrivalId: ArrivalId): Action[AnyContent] =
-    actions.requireData(arrivalId) {
-      Redirect(
-        controllers.routes.NewAuthYesNoController.onPageLoad(arrivalId, NormalMode)
-      )
+    actions.requireData(arrivalId).async {
+      implicit request =>
+        val numberOfSeals = request.userAnswers.ie043Data.Consignment.map(_.TransportEquipment.map(_.numberOfSeals.intValue).sum).getOrElse(0)
```

Comment:
Annoyingly, seals actually exist in two parts of the IE043. Transport equipment and incidents. There is a helper method in RichCC043CType (`sealsExist`) that could be used here instead to cover those two bases.